test/commands.test.js

```diff
@@ -621,7 +622,12 @@ test('[commands.operations.getMasterConfig] success', function(assert) {
     }
   });
 
-  context.oldParameters = { old: 'stale' };
+  AWS.stub('KMS', 'decrypt', function(params, data) {
+    return 'data';
+  });
+
+  context.oldParameters = { old: 'secure:staleelats' };
+  context.overrides = { templateOptions : { AWS_DEFAULT_REGION : 'us-east-1'}};
```

Comment:
Good question -- the `AWS_DEFAULT_REGION` comes from `templateOptions.AWS_DEFAULT_REGION`, but I'm not sure if this change is not generalizable enough for cfn-config.